### PR TITLE
Blackjack

### PR DESCRIPTION
**Pull request by danielyxie**

### CODE REVIEW

File: src/Casino/Blackjack.tsx [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+        } = this.state;
+
+        // Get the player totals to display. 
+        const playerHandValues = this.getHandValue(playerHand);
+        if (playerHandValues[1] > 21 || playerHandValues[0] === playerHandValues[1]) { playerHandValues.pop() }
+
+        const dealerHandValues = this.getHandValue(dealerHand);
+        if (dealerHandValues[1] > 21 || dealerHandValues[0] === dealerHandValues[1]) { dealerHandValues.pop(); }
+
+        console.log(dealerHandValues);
```
Comment (hydroflame) [inline src/Casino/Blackjack.tsx:266]: I'd be nice to remove those.

**Comment by hydroflame** [comment]
Screen Shot 2021-04-23 at 3 07 19 AM

**Comment by hydroflame** [comment]
Buttons need isTrusted safeguards.

**Comment by hydroflame** [comment]
Screen Shot 2021-04-23 at 3 09 44 AM

**Comment by hydroflame** [comment]
Screen Shot 2021-04-23 at 3 13 13 AM

**Comment by hydroflame** [comment]
Screen Shot 2021-04-23 at 3 15 30 AM


The dealer doesn't try to tie?

**Comment by danielyxie** [comment]
Addressed all feedback.

As for the last one about dealer trying to win...I went with Vegas rules and I think for Vegas rules dealers will never hit once you hit 17?

**Comment by sschmidTU** [comment]
> for Vegas rules dealers will never hit once you hit 17?

looks like it.
"The dealer has to hit his hand when he has 16 or less and stand (not take more cards) on 17 or better."
https://www.lasvegasdirect.com/las-vegas-blackjack-rules-how-to-play-blackjack-and-win/

**Pull request merged**